app/invocation/invocation_model.tsx

```diff
@@ -514,6 +515,16 @@ export default class InvocationModel {
     );
   }
 
+  getFiles(event: build_event_stream.IBuildEvent): build_event_stream.IFile[] {
+    if (!event?.completed) {
+      return [];
+    }
+    if (event.completed.importantOutput?.length) {
+      return event.completed.importantOutput;
+    }
+    return event.completed.outputGroup?.flatMap((group) => group.fileSets).flatMap((set) => this.files.get(set.id));
```

Comment:
nit: maybe change to `this.files.get(set.id) || []` just to be safe?

Reply:
Done

Reply:
Sry I meant like

```
.flatMap((set) => this.fileSetIDToFilesMap.get(set.id) || [])
```

(`.get()` returns undefined if the entry isn't found in the map)

Reply:
Will send fix in another PR